releng/org.eclipse.xtext.dev-bom/pom.xml

```diff
@@ -0,0 +1,308 @@
+				<groupId>org.eclipse.platform</groupId>
+				<artifactId>org.eclipse.core.jobs</artifactId>
+				<version>${core-jobs-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.platform</groupId>
+				<artifactId>org.eclipse.core.resources</artifactId>
+				<version>${core-resources-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.platform</groupId>
+				<artifactId>org.eclipse.core.runtime</artifactId>
+				<version>${core-runtime-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.platform</groupId>
+				<artifactId>org.eclipse.equinox.app</artifactId>
+				<version>${equinox-app-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.platform</groupId>
+				<artifactId>org.eclipse.equinox.common</artifactId>
+				<version>${equinox-common-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.platform</groupId>
+				<artifactId>org.eclipse.equinox.preferences</artifactId>
+				<version>${equinox-preferences-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.platform</groupId>
+				<artifactId>org.eclipse.equinox.registry</artifactId>
+				<version>${equinox-registry-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.jdt</groupId>
+				<artifactId>org.eclipse.jdt.compiler.apt</artifactId>
+				<version>${jdt-compiler-apt-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.jdt</groupId>
+				<artifactId>org.eclipse.jdt.compiler.tool</artifactId>
+				<version>${jdt-compiler-tool-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.jdt</groupId>
+				<artifactId>org.eclipse.jdt.core</artifactId>
+				<version>${jdt-core-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.platform</groupId>
+				<artifactId>org.eclipse.osgi</artifactId>
+				<version>${osgi-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.platform</groupId>
+				<artifactId>org.eclipse.text</artifactId>
+				<version>${text-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.xpand</groupId>
+				<artifactId>org.eclipse.xpand</artifactId>
+				<version>${xpand-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.xpand</groupId>
+				<artifactId>org.eclipse.xtend</artifactId>
+				<version>${xpand-version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.xpand</groupId>
+				<artifactId>org.eclipse.xtend.typesystem.emf</artifactId>
+				<version>${xpand-version}</version>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+
+	<build>
+		<plugins>
+			<plugin>
+				<artifactId>maven-resources-plugin</artifactId>
+				<version>3.1.0</version>
+				<executions>
+					<execution>
+						<id>copy-resource-one</id>
+						<phase>install</phase>
+						<goals>
+							<goal>copy-resources</goal>
```

Comment:
Why is this here?

Reply:
This copies the BOM during the `install` phase to the target location that is archived . This avoids the misuse of the `deploy` goal and would allow to use the deploy goal for a real deployment to a remote repo.